compiler/resolution/implementForallIntents.cpp

```diff
@@ -994,7 +995,17 @@ static ShadowVarSymbol* createSVforFieldAccess(LoopWithShadowVarsInterface* fs,
                                                Symbol* field)
 {
   bool isConst = ovar->isConstant() || field->isConstant();
-  VarSymbol* fieldRef = createFieldRef(fs->asExpr(), ovar, field, isConst);
+
+  Expr* anchor = nullptr;
```

Comment:
Could you add a comment explaining what's happening here. (or at least indicate that it's a patch to workaround an issue that comes up when using `--baseline`.

Reply:
https://github.com/chapel-lang/chapel/pull/24556/commits/ae5576b4afda65dd0954e46786d5d044e109c44b